core/src/Platforms/Android/EmbeddedWebview/EmbeddedWebUI.cs

```diff
@@ -76,7 +76,9 @@ public new static void SetAuthorizationResult(AuthorizationResult authorizationR
             }
             else
```

Comment:
>else [](start = 12, length = 4)

would it make sense to raise an exception instead allowing the developer to do something about this situation?

Reply:
i removed this...but i'll do a quick test in the morning before merging. nothing seems to be using this code. Looks like it's a duplicate method and should have been deleted here, as it's used elsewhere.

---
In reply to: [206366224](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1154#discussion_r206366224) [](ancestors = 206366224)